Content.Shared/Maps/ContentTileDefinition.cs

```diff
@@ -83,6 +83,12 @@ public sealed partial class ContentTileDefinition : IPrototype, IInheritingProto
         /// </summary>
         [DataField("isSpace")] public bool MapAtmosphere { get; private set; }
 
+        /// <summary>
+        /// Denotes whether the tile is the station lattice
+        /// </summary>
+        [DataField]
+        public bool IsLattice { get; private set; }
+
```

Comment:
Doesn't this duplicate the IsSpace property?

Reply:
Yup, you're right. Removed this field